app/models/member.rb

```diff
@@ -120,7 +123,7 @@ def access_level_in_namespace_group_links(user, namespace)
 
       if namespace_group_links.count.positive?
         maxlevel_namespace_group_link = namespace_group_links.order(:group_access_level).last
-        membership = Member.for_namespace_and_ancestors(maxlevel_namespace_group_link&.group)
+        membership = Member.for_namespace_and_ancestors(maxlevel_namespace_group_link&.group).not_expired
                      &.where(user:)&.order(:access_level)
 
         return [maxlevel_namespace_group_link.group_access_level, membership.last.access_level].min
```

Comment:
We only want to return here if memberships exist from line 126. Otherwise this will throw an error when trying to do a `min`. 
To test this out you can create a new user, as `admin@email.com` add this user to `Bacillus`. Logout and login with this new user. Try to visit group `Yersinia`. You will see the error is caused by the `membership.last` in `membership.last.access_level`

Reply:
We still need to use the `.not_expired` when getting the memberships. The return where we get the minimum access level of the two needs to be wrapped in an if statement which checks if this membership array has a count

Reply:
f33e5a21dec476898a956cf201c0cbeb26e57744